src/js/services/forms.js

```diff
@@ -48,6 +51,8 @@ export default App.extend({
   },
   readyForm() {
     this.trigger('ready');
+
+    this.refreshForm();
```

Comment:
I initially put this in the `initialize()` function. But if a user is prompted with the keep/discard work prompt, `this.refreshForm()` would be called before the actual form has been shown to the user.

So I put it here instead. To wait until the form content is loaded.